recipes-core/shellhub/shellhub-agent_0.9.1.bb

```diff
@@ -35,6 +35,7 @@ GO_LDFLAGS = '-ldflags="${GO_RPATH} ${GO_LINKMODE} -X main.AgentVersion=v${PV} -
 GOBUILDFLAGS:append = " -modcacherw"
 
 do_compile[dirs] += "${B}/src/${GO_IMPORT}/agent"
+do_compile[network] = "1"
```

Comment:
Does it fails otherwise? It shouldn't.

Reply:
Yes. with the network does not enable during the compile task the shellhub doesn't compile. An error occurs to access the golang proxy.
`proxy.golang.org on 10.5.3.1:53: dial udp 10.5.3.1:53: connect: network is unreachable `

Reply:
Ok.